packages/rspack/src/config/zod/builtins.ts

```diff
@@ -1,6 +1,28 @@
 import { z } from "zod";
 
 export function builtins() {
-	// TODO(hyf0): need to enable strict mode when developer have time to write these schema
-	return z.record(z.any());
+	// TODO(hyf0): need to use `strictObject` mode when developer have time to finish the whole schema
+	return z.object({
+		postcss: z
+			.strictObject({
+				pxtorem: z
+					.strictObject({
+						rootValue: z.number().optional(),
+						unitPrecision: z.number().optional(),
+						selectorBlackList: z.string().array().optional(),
+						propList: z.string().array().optional(),
+						replace: z.boolean().optional(),
+						mediaQuery: z.boolean().optional(),
+						minPixelValue: z.number().optional()
+					})
+					.optional()
+			})
+			.refine(() => {
+				console.warn(
```

Comment:
It will be better to use https://nodejs.org/api/util.html#utildeprecatefn-msg-code